client/src/components/SearchBar.js

```diff
@@ -92,6 +93,7 @@ class SearchBar extends Component {
 	onSubmit(event) {
 		if (!this.state.showAdvancedSearch) {
 			// We only update the Redux state on submit
+			this.props.resetPages()
```

Comment:
I think this is why now we have the following problem:
- I search on positions (also add some filters on positions)
- I go to the search bar, change the search term and click search
- the search filter is gone now

Reply:
Should be fixed